### PR TITLE
New styles for site home

### PR DESCRIPTION
**Pull request by thisisdano**
**A pair to https://github.com/18F/cg-site/pull/661 — these should go live together.**

Edits to style to accommodate new homepage copy needs.

cg-homepage-changes

### CODE REVIEW

File: src/css/base/typography.scss [hunk 1]
```diff
@@ -144,56 +144,57 @@ p {
...
+    a {
+      border-bottom: 1px solid;
+      padding-bottom: 2px;
+      text-decoration: none;
 
-    &:visited {
-      border-bottom-style: dotted;
+      &:visited {
+        border-bottom-style: dotted;
+      }
```
Comment (line47) [inline src/css/base/typography.scss:186]: The visited border style isn't working locally.

Comment (thisisdano) [reply src/css/base/typography.scss:186]: never got it to work, dunno why!

Comment (line47) [reply src/css/base/typography.scss:186]: Border-style is not supported for visited. https://developer.mozilla.org/en-US/docs/Web/CSS/:visited

Comment (line47) [reply src/css/base/typography.scss:186]: I think this should be removed for now.

Comment (thisisdano) [reply src/css/base/typography.scss:186]: ha, ok! that's strange...

Comment (thisisdano) [reply src/css/base/typography.scss:186]: We'll let the nsa slurp all our personal communication out of telecom servers, but not put a bordered bottom rule on a `:visited` pseudoelement for privacy reasons.

**Review by line47 — CHANGES_REQUESTED** [review]

File: src/css/base/typography.scss (hunk 1 above)
Comment (thisisdano) [reply src/css/base/typography.scss:186]: OK, changed it to just a gray line.

**Review by line47 — APPROVED** [review]

**Pull request merged**